### PR TITLE
Desktop privacy section

### PR DESCRIPTION
**Pull request by mindmonk**
A privacy section has been introduced in the Desktop documentation, elucidating data processing during error handling.

### CODE REVIEW

File: source/desktop/privacy.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+The error database is hosted on our server, so all requests are directed exclusively to it.
+Without your explicit consent, no requests will be executed, as this feature operates solely on an Opt-In basis.
+
+Simply click the ``Look up Solution`` button.
+
+We will cross-reference your current error with the database and provide a solution link if one is available.
+
+.. ToDo:
+   Add "More info..." hyperlink to error dialog and refresh screenshot.
+
```
Comment (SailReal) [inline source/desktop/privacy.rst:21]:
```suggestion
```

File: source/desktop/privacy.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Error support
+-----------------------------
+
+Cryptomator is primarily developed with privacy concerns in mind, and we have not compromised on this principle when introducing this feature.
+As part of our continuous software improvement process, we employ this feature to provide users with available solutions, identify potential error clusters, and manage them with prioritization, all while ensuring the preservation of user privacy.
+
+
+Enhancing error management and facilitating problem resolution, we offer you the option to access our up-to-date error database.
+The error database is hosted on our server, so all requests are directed exclusively to it.
+Without your explicit consent, no requests will be executed, as this feature operates solely on an Opt-In basis.
```
Comment (SailReal) [inline source/desktop/privacy.rst:13]:
```suggestion
Without your explicit consent, no requests will be executed, as this feature operates solely on an opt-in basis.
```

**Review by SailReal — CHANGES_REQUESTED** [review]
Just some minor hints...

Is it also planned to add a sentence about the "Look up solution" button?

**Comment by tobihagemann** [comment]
I have a much broader question. Are we sure that we want to add a privacy section in the docs? Shouldn't it be part of the privacy policy under section 8 on our website? https://cryptomator.org/privacy/#app

**Comment by mindmonk** [comment]
I agree with your comment. Accordingly, I am closing the pull request.

**Pull request closed without merging**